src/main/java/org/dataone/hashstore/filehashstore/FileHashStore.java

```diff
@@ -1458,6 +1670,21 @@ private void isStringEmpty(String string, String argument, String method) {
         }
     }
 
+    /**
+     * Checks whether a given long object is greater than 0
+     *
+     * @param object Object to check
```

Comment:
it isn't an object :-)

Reply:
D'oh - updated, thanks for pointing this out!